graphql/src/store/resolver.rs

```diff
@@ -76,12 +76,9 @@ impl StoreResolver {
     ) -> Result<Self, QueryExecutionError> {
         let store_clone = store.cheap_clone();
         let deployment2 = deployment.clone();
-        let block_ptr = graph::spawn_blocking_allow_panic(move || {
-            Self::locate_block(store_clone.as_ref(), bc, deployment2.clone())
-        })
-        .await
-        .map_err(|e| QueryExecutionError::Panic(e.to_string()))
-        .and_then(|x| x)?; // Propagate panics.
+        let block_ptr = Self::locate_block(store_clone.as_ref(), bc, deployment2.clone())
+            .await
+            .map_err(|e| QueryExecutionError::Panic(e.to_string()))?;
```

Comment:
```suggestion
            ?;
```
No need for error gymnastics now that the spawn_blocking is gone.